src/app/modules/main/browser_section/dapps/accounts.nim

```diff
@@ -0,0 +1,91 @@
+import NimQml, Tables, strutils, stew/shims/strformat
+import ../../../../../app_service/service/wallet_account/service as wallet_account_service
+
+type
+  ModelRole {.pure.} = enum
+    Name = UserRole + 1
+    Address
+    ColorId
+    Emoji
+
+QtObject:
+  type
+    AccountsModel* = ref object of QAbstractListModel
+      items: seq[WalletAccountDto]
+
+  proc delete(self: AccountsModel) =
+    self.items = @[]
+    self.QAbstractListModel.delete
+
+  proc setup(self: AccountsModel) =
+    self.QAbstractListModel.setup
+
+  proc newAccountsModel*(): AccountsModel =
+    new(result, delete)
+    result.setup
+
+  proc `$`*(self: AccountsModel): string =
+    for i in 0 ..< self.items.len:
+      result &= fmt"""
+      [{i}]:({$self.items[i].name})
+      """
+
+  proc modelChanged(self: AccountsModel) {.signal.}
+  proc countChanged(self: AccountsModel) {.signal.}
```

Comment:
Again two useless signals. `modelChanged` doesn't seem used. The other one is most likely used, but we can replace by ModelCount